monkeytype/stubs.py

```diff
@@ -503,22 +503,62 @@ def __repr__(self) -> str:
                                           tuple(self.function_stubs.values()),
                                           tuple(self.attribute_stubs))
 
-    @staticmethod
-    def stubs_from_typed_dict(type_dict: type, class_name: str) -> List['ClassStub']:
-        """Return a list of class stubs for all TypedDicts found within `type_dict`."""
-        assert is_anonymous_typed_dict(type_dict)
-        class_stubs = []
+
+class ReplaceTypedDictsWithStubs(TypeRewriter):
+    """Replace TypedDicts in a generic type with class stubs and store all the stubs."""
+
+    def __init__(self, class_name_hint: str) -> None:
+        self._class_name_hint = class_name_hint
+        self._stubs: List[ClassStub] = []
+
+    @property
+    def stubs(self) -> List[ClassStub]:
+        return self._stubs
```

Comment:
What's the purpose of this indirection?

Reply:
Nothing :) I've made `stubs` a public attribute now.